### PR TITLE
Use singular http clients

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

**Review by j16r — DISMISSED** [review]

File: core/utils/http.go [hunk 1]
```diff
@@ -13,6 +13,24 @@ import (
 	"github.com/smartcontractkit/chainlink/core/logger"
 )
 
+var (
+	Client             *http.Client
```
Comment (connorwstein) [inline core/utils/http.go:17]: do we need to export these?

Comment (samsondav) [reply core/utils/http.go:18]: I was hoping to re-use them in this PR: https://github.com/smartcontractkit/chainlink/pull/3859

File: core/utils/http.go [hunk 2]
```diff
@@ -6,13 +6,47 @@ import (
...
+			KeepAlive: 30 * time.Second,
+			DualStack: true,
+		}).DialContext,
+		ForceAttemptHTTP2:     true,
+		MaxIdleConns:          100,
+		IdleConnTimeout:       90 * time.Second,
+		TLSHandshakeTimeout:   10 * time.Second,
+		ExpectContinueTimeout: 1 * time.Second,
+	}
+	t.DisableCompression = true
```
Comment (connorwstein) [inline core/utils/http.go:37]: hm why not just inside the literal def?

**Review by connorwstein — APPROVED** [review]

File: core/utils/http.go (hunk 2 above)
Comment (samsondav) [reply core/utils/http.go:37]: Because the top part is copied verbatim from the golang docs.

DisableCompression is our addition

**Review by connorwstein — APPROVED** [review]

**Pull request merged**